src/Plugin.php

```diff
@@ -143,6 +143,11 @@ public static function init() {
       add_filter('woocommerce_email_styles', __NAMESPACE__ . '\WooCommerce::woocommerce_email_styles');
     }
 
+    if (get_option('_add_salutation_field') === 'yes') {
```

Comment:
Missing plugin namespace prefix.

Reply:
While I agree, I oriented on the already existing option names which don't use a prefix as well. If we add a prefix, we should add it to the other options as well.